src/main/java/com/andersen/hw/model/User.java

```diff
@@ -2,13 +2,27 @@
 
 import com.andersen.hw.util.IdGenerator;
 
+import java.time.LocalDateTime;
+
 abstract public class User implements Identifiable, Printable {
-    private final int classId;
+    private String name;
+    private Long userId;
+    private LocalDateTime creationDate;
+    private int classId;
```

Comment:
remove userId and classId. You need to have just 1 autogenerated id for each instance of User
```java
private int id
```